### ISSUE TITLE
Add vignette skeleton to document randomization ratio behaviors

### ISSUE THREAD

**Issue by nanxstats** [Merck/simtrial#283]
Randomization ratio seems to be an important and complex enough input (or inferred) parameter that warrants somewhat detailed documentation.

I will create a vignette skeleton to get things started, with the diagram.

**Comment by LittleBeannie** [comment]
Hi @nanxstats, do you have a timeline to get this finished?

**Comment by nanxstats** [comment]
Nope, apparently this has not been able to make priorities list so I will close this as won't do.

In case someone is interested, some relevant information in #281.
